### PR TITLE
Remove 'Coming Soon' results from report

### PR DESCRIPTION
**Pull request by paulirish**
Say goodbye to these friends:

image

image

ref #332, #333

### CODE REVIEW

**Comment by brendankenny** [comment]
:parade:

**Comment by paulirish** [comment]
#### As for followup issues:

> "Content scrolls at 60fps"
> "Touch input gets a response in < 150ms"

These, along with warm reload, are longstanding features on the backlog. No issue filed for them, but they are on the periphery.

> "App is interactive without jank after the first meaningful paint"

I think this was EIL, which we actually delivered. We could report ranges of EIL  -- such as the EIL from FCP (or FMP) to TTI. And from TTI to the end.  But I think those numbers will be of limited usefulness without a visualization (like what webpagetest does: 
image )


> "Service worker makes use of push notifications, if appropriate"

theres a huge section about this on the [Progressive Web App Checklist](https://developers.google.com/web/progressive-web-apps/checklist#site-appropriately-informs-the-user-when-theyre-offline) that discusses how to do it right. we can't detect nearly any of that, so I'm supportive of deferring to this document.


> "Tap targets are appropriately sized for touch"

captured in https://github.com/GoogleChrome/lighthouse/issues/909

> "Payment forms marked up with `autocomplete` attributes"
> "Login forms marked up with `autocomplete` attributes"
> "Input fields use appropriate `type` attributes for custom keyboards"

I'll file a ticket for these three now.

**Comment by ebidel** [comment]
Someone was recently asking for a tracking bug for "Scrolls at 60fps". Do you want to file that?

File: lighthouse-core/report/templates/report-template.html [hunk 1]
```diff
@@ -187,9 +183,7 @@ <h2 class="config-section__title">Blocked URL Patterns</h2>
                   {{/if}}
 
                   <div class="subitem-result">
-                    {{#if subItem.comingSoon}}
-                          <span class="subitem-result__unknown score-unknown-bg">N/A</span>
-                    {{else if subItem.error}}
+                    {{#if subItem.error}}
```
Comment (brendankenny) [inline lighthouse-core/report/templates/report-template.html]: maybe remove `.subitem-result__unknown.` and `.score-unknown-bg` now that they're unused? are there other things we're missing?

Comment (brendankenny) [inline lighthouse-core/report/templates/report-template.html]: Need to run the css usage gatherer against report :)

Comment (ebidel) [inline lighthouse-core/report/templates/report-template.html]: We still need the unknown error background image. https://github.com/GoogleChrome/lighthouse/commit/a067d3060b4fb79ebb6ca11e412c2c12699107f3#diff-181e527364f32304fefacb83683da1a7R193.

Comment (brendankenny) [inline lighthouse-core/report/templates/report-template.html]: oh right :) so just `score-unknown-bg`?

**Comment by paulirish** [comment]
@ebidel filed scrolling issue: https://github.com/GoogleChrome/lighthouse/issues/1648

File: lighthouse-core/report/templates/report-template.html (hunk 1 above)
Comment (paulirish) [inline lighthouse-core/report/templates/report-template.html]: nuked that rule.


also here's the unused CSS against our report: 
image

haha

**Comment by brendankenny** [comment]
screen shot 2017-02-06 at 17 49 32

**Pull request merged**